### PR TITLE
Update stale.yml to not close PR if someone takes 1 week of vacation

### PR DESCRIPTION
**Pull request by ldemailly**
1 week of inactivity isn't enough - people take breaks; holidays etc...

### CODE REVIEW

**Comment by cmluciano** [comment]
@ldemailly dupe of #6507 ?

File: .github/stale.yml [hunk 1]
```diff
@@ -7,14 +7,14 @@ staleLabel: stale
 # Pull request specific configuration
 pulls:
   # Number of days of inactivity before an Issue or Pull Request becomes stale
-  daysUntilStale: 7
+  daysUntilStale: 14
   # Number of days of inactivity before a stale Issue or Pull Request is closed.
   # Set to false to disable. If disabled, issues still need to be closed manually, but will remain marked as stale.
-  daysUntilClose: 7
```
Comment (hklai) [inline .github/stale.yml]: I think 30 days and 14 days freshness for issues and prs are good, but if nobody cares about something becoming state, why do we need to wait for another week to close it?

File: .github/stale.yml [hunk 2]
```diff
@@ -30,14 +30,14 @@ issues:
...
-    last 30 days. It will be closed in the next 7 days unless it is tagged "help wanted" or other activity
+    last 30 days. It will be closed in the next 2 weeks unless it is tagged "help wanted" or other activity
     occurs. Thank you for your contributions.
   closeComment: >
     This issue has been automatically closed because it has not had activity in the
-    last 37 days. If this issue is still valid, please ping a maintainer and ask them to label it as "help wanted".
+    last month and a half. If this issue is still valid, please ping a maintainer and ask them to label it as "help wanted".
     Thank you for your contributions.
   exemptLabels:
     - help wanted
```
Comment (hklai) [inline .github/stale.yml]: should avoid making customer issues as stale. Should prob add the "kind/customer issue" label here.

**Comment by sebastienvas** [comment]
I started https://github.com/istio/istio/pull/6507 yesterday. Feel free to combine changes.

File: .github/stale.yml (hunk 1 above)
Comment (ldemailly) [reply .github/stale.yml]: what would happen is people forgot about a PR and they need the bot ping to remind them; I suppose they could reopen too but... some notice is good imo

**Pull request merged**